### PR TITLE
David/feature/ping2.0

### PR DESCRIPTION
**Pull request by SonDav**
I refactored the ping.js and user.js code to v14. Simple changes.

### CODE REVIEW

File: src/.env_template
Comment (michaelFarrier) [inline src/.env_template:1]: This file should not be deleted.

File: src/commands/util/ping.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+		
+		const pingEmbed = new EmbedBuilder()
+			.setColor('#4fffa4')
+			.setTitle('🏓 Pong!')
+			.addFields(
+				{name: 'Bot Latency', value: `**${botLatency}ms**`},
+				{name: 'API Latency', value: `**${apiLatency}ms**`},
+			)
+
+		return await interaction.editReply({
```
Comment (michaelFarrier) [inline src/commands/util/ping.js:28]: remove return & re-test w/o the return it should function the same

File: src/.env_template
Comment (SonDav) [reply src/.env_template:1]: I didn't delete the file I just renamed it.

File: src/commands/util/user.js [hunk 2]
```diff
@@ -1,14 +1,83 @@
...
+		// Grab all the guild member's roles
+		const guildMemberRolesSize = guildMember.roles.cache.size - 1;
+		const guildMemberRoles = guildMember.roles.cache.sort((r1, r2) => r2.rawPosition - r1.rawPosition);
+		const guildMemberRolesText = guildMemberRoles
+			.map((r) => r)
+			.slice(0, -1)
+			.join(", ");
+
+		// Grab all the guild member's award roles
+		const guildMemberAwards = guildMember.roles.cache
```
Comment (michaelFarrier) [inline src/commands/util/user.js:49]: Remove awards logic since we do not use awards anymore

File: src/commands/util/ping.js (hunk 1 above)
Comment (michaelFarrier) [inline src/commands/util/ping.js:28]: remove return & re-test w/o the return it should function the same

File: src/.env_template
Comment (michaelFarrier) [inline src/.env_template:1]: This file should not be deleted.

File: src/commands/util/ping.js (hunk 1 above)
Comment (SonDav) [reply src/commands/util/ping.js:28]: Did not work... have to keep the return.

File: src/commands/util/user.js [hunk 3]
```diff
@@ -1,14 +1,83 @@
...
+					value: guildMemberJoined,
+					inline: true,
+				},
+				{
+					name: "💎 Booster",
+					value: guildMemberPremium ? guildMemberPremium : "Not a booster",
+					inline: true,
+				})
+            .addFields({name: `🎭 All Roles (${guildMemberRolesSize})`, value: guildMemberRolesText ? guildMemberRolesText: "No roles assigned"})
+            .addFields({name: `🏆 Award Roles (${guildMemberAwardsSize})`, value: guildMemberAwardsText ? guildMemberAwardsText : "No awards"})
```
Comment (michaelFarrier) [inline src/commands/util/user.js:79]: remove awards logic see above comment

File: src/commands/util/user.js [hunk 4]
```diff
@@ -1,14 +1,83 @@
...
+	
+		return await interaction.reply({ embeds: [userInfoEmbed] });
```
Comment (michaelFarrier) [inline src/commands/util/user.js:81]: remove return and re-test

**Review by michaelFarrier — CHANGES_REQUESTED** [review]

File: src/commands/util/user.js (hunk 2 above)
Comment (SonDav) [reply src/commands/util/user.js:49]: Ok done.

File: src/.env_template
Comment (SonDav) [reply src/.env_template:1]: I KEEP TELLING YOU I DIDN'T TOUCH IT

File: src/commands/util/user.js (hunk 3 above)
Comment (SonDav) [reply src/commands/util/user.js:79]: Yes

File: src/commands/util/ping.js (hunk 1 above)
Comment (SonDav) [reply src/commands/util/ping.js:28]: I already told you it DOESN'T WORK!

File: src/commands/util/user.js (hunk 4 above)
Comment (SonDav) [reply src/commands/util/user.js:81]: Ok this one works

Comment (SonDav) [reply src/commands/util/user.js:81]: Scratch that I removed from the wrong file.

Comment (SonDav) [reply src/commands/util/user.js:81]: YEA IT DOESN'T WORK! STOP ASKING ME!

File: src/commands/util/ping.js (hunk 1 above)
Comment (SonDav) [reply src/commands/util/ping.js:28]: Hey uh turns out you were correct hee hee xD

File: src/commands/util/user.js (hunk 4 above)
Comment (SonDav) [reply src/commands/util/user.js:81]: Hey uhhh so it works... uh my bad hehexd

Comment (ianchalk) [reply src/commands/util/user.js:81]: lmao

**Comment by michaelFarrier** [comment]
just start over man

**Pull request closed without merging**